wormhole/wormhole_test.go

```diff
@@ -247,46 +253,54 @@ func TestWormholeFileTransportSendRecvViaRelayServer(t *testing.T) {
 
 	url := rs.WebSocketURL()
 
-	relayServer := newTestTCPRelayServer()
-	defer relayServer.close()
+	testDisableLocalListener = true
+	defer func() { testDisableLocalListener = false }()
```

Comment:
This might not work anymore on our master as we removed this global variable in 66d2eb99.

Reply:
Should this code be updated to use `t.disableListener`, reverted to the old revision, removed entirely... I see [lines 250-254](https://github.com/LeastAuthority/wormhole-william/commit/66d2eb99a73f07a7d564f8c999563cec827da672#diff-a5f4908597678c5fcd9c56291d6ac3c70c0ff0102b3d592bdd95dbd8900df534L250-L254) preserves the old code so I assume we can just undo this change?

Reply:
Oh never mind, functions such as Receive() set that already. So I guess that leaves the question, do we want the old relayServer assignment and deferment?